Cargo.toml

```diff
@@ -3,10 +3,10 @@ name = "socket-server-mocker"
 description = "Mock socket server in Rust, for testing various network clients."
 authors = ["Thomas Prévost"]
 version = "0.1.1"
+rust-version = "1.60.0"
```

Comment:
Why ? Could you justify please? I expect this crate working for any version of Rust

Reply:
try [cargo msrv](https://github.com/foresterre/cargo-msrv) tool - it will tell you the oldest version your crate will support.  Note that you don't want to support too many versions back - the more you support, the fewer language things you can use.

Simple examples:
* `format!("foo = {foo}")` is only available after [1.58](https://releases.rs/docs/1.58.0/)
* `edition=2021` is only available after [1.56](https://releases.rs/docs/1.56.0/)
* `let Some(v) = value else {return;}` is even more recent, etc.

Reply:
interestingly, MSRV 1.60 cannot even be tested because your dev dependencies require 1.70+ - so might as well set MSRV to 1.70+.  TBH, in Rust community very few crates support even 10 versions back -- that's ~1.3 years back.  Or you may want to change the way you test, e.g. have a "basic test mode" that does not require any additional dependencies.  You still will have to deal with the fact that many language and cargo features you would want are only available for the last 1-2 years

Reply:
@nyurik it makes the build fail apparently